srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
```

Comment:
Arguably the correct format would be `NoiseTorch`... But staying with this is valid.